src/main/java/org/dasein/cloud/test/network/StatefulVPNTests.java

```diff
@@ -131,13 +129,13 @@ public void createInternalVPN() throws CloudException, InternalException {
 
             for (VPNProtocol protocol : supportedProtocols) {
                 tm.out("Testing VPN protocol: " + protocol);
```

Comment:
I'll have more comments about this test, but let's start from this loop. Do we need to test all supported protocols? For our intents one should be well enough: `protocol = supportedProtocols.iterator().next();`